internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -665,6 +786,41 @@ func expandAppConfigurationEncryption(input []interface{}) *configurationstores.
 	return result
 }
 
+func expandAppConfigurationReplicas(input []interface{}, configurationStoreName, configurationStoreLocation string) ([]replicas.Replica, error) {
+	result := make([]replicas.Replica, 0)
+
+	locationSet := map[string]interface{}{
+		configurationStoreLocation: nil,
+	}
+	replicaNameSet := map[string]interface{}{}
+
+	for _, v := range input {
+		replica := v.(map[string]interface{})
+		replicaName := replica["name"].(string)
+		replicaLocation := location.Normalize(replica["location"].(string))
+
+		if _, ok := locationSet[replicaLocation]; ok {
+			return result, fmt.Errorf("replica location %q is duplicated in configuration store %q location", replicaLocation, configurationStoreName)
+		}
+		locationSet[replicaLocation] = nil
```

Comment:
Same for this line, why are we setting this to nil?

Reply:
as above one